pub/assets/js/hound.js

```diff
@@ -284,12 +284,13 @@ var Model = {
   UrlToRepo: function(repo, path, line) {
     var info = this.repos[repo],
         url = info.url.replace(/\.git$/, ''),
-        anc = line ? '#L' + line : '';
+        anc = line ? info.anchorpattern.replace('${line}', line) : '';
```

Comment:
Yes, if the server does not provide anchorpattern or urlpattern this will break (same as url).
So this change breaks downwards compatibility, the new front end will break with an old backend.